utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,61 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.LevelListDrawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import java.net.URL
+
+/** UrlImage Parser for android TextView to load Html Image tag. */
+class UrlImageParser(internal var tvContents: TextView, internal var context: Context) : Html.ImageGetter {
```

Comment:
Can you add dedicated tests for this parser? If it makes things easier, consider sending out a PR first with the parser & its tests to get that checked in right away, and then you can base off of that branch to complete the rest of this.

Reply:
Separate PR #201 is been made for UrlImageParser.